EpsilonAuxBmsV2/Tests/tests/SendAuxTripTaskTest/SendAuxTripTaskTest.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+
+int runSendAuxTripTaskTest();
```

Comment:
Make sure all your files have a new line at the end

Reply:
Only .c or .h files tho. You don't need new lines for your Makefiles

Reply:
Ohhh I see